cmd/zdb/zdb.c

```diff
@@ -21,11 +21,12 @@
 
```

Comment:
shouldn't this become a separate PR? that's not zstd-related, is it?

Reply:
Shoot, let me take a look... afaik there was a reason.

Reply:
Ahh i'll remove this one.
I originally added Allan's tag, but in hindsight it was not a copyrightable change...